packages/html-reporter/src/testResultView.tsx

```diff
@@ -106,6 +106,10 @@ export const TestResultView: React.FC<{
       </AutoChip>
     )}
 
+    {!!result.steps.length && <AutoChip header='Test Steps'>
```

Comment:
Why do we show this twice now?

Reply:
Looks like a problem with my git patch. Removed.